src/tools/admin/mise-en-relation.js

```diff
@@ -119,23 +119,65 @@ execute(__filename, async ({ db }) => {
     }]).toArray();
```

Comment:
@jystervinou En local j'ai une erreur de remonté à propos du num 
`$geoNear no longer supports the 'num' parameter. Use a $limit stage instead`

Reply:
erreur ou warning ?

Reply:
quelle version de mongo ?

Reply:
erreur et mongo 4.4.22